src/constants/colors.css

```diff
@@ -0,0 +1,11 @@
+:root {
+  --theme-color: #efefef;
+  --primary-color: #000e29;
+  --secondary-color: #333;
```

Comment:
minor but still: try to be consistent and use #333333 instead

Reply:
fixed